tests/ci/cdk/README.md

```diff
@@ -58,27 +58,30 @@ To setup or update the CI in your account you will need the following IAM permis
   * secretsmanager:DeleteSecret
   * secretsmanager:GetSecretValue
 
-### Command
+### Commands
+
+Note: `GITHUB_REPO_OWNER` is to specify the GitHub repo targeted by this CI setup.
```

Comment:
Nit: Note: `GITHUB_REPO_OWNER` specifies the GitHub repo targeted by this CI setup.

Reply:
Addressed.